x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout/jump_to_date.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonEmpty, EuiDatePicker } from '@elastic/eui';
+import React, { forwardRef } from 'react';
+import moment from 'moment';
+
+interface Props {
+  selectedDate: moment.Moment | null;
+  onChangeSelectedDate: (date: moment.Moment | null) => void;
+  onClick?: () => void;
+  value?: string;
+}
+
+export const JumpToDate: React.FunctionComponent<Props> = (props) => {
+  return (
+    <EuiDatePicker
+      data-test-subj="agentActivityFlyout.jumpToButton"
+      selected={props.selectedDate}
+      onChange={props.onChangeSelectedDate}
+      maxDate={moment()}
+      customInput={<CustomInput {...props} />}
+    />
+  );
+};
+
+interface CustomInputProps {
+  onClick?: () => void;
+  value?: string;
+}
+
+const CustomInput = forwardRef(({ onClick, value }: CustomInputProps, ref) => {
```

Comment:
nit: Should we come up with a more meaningful name than CustomInput?

Reply:
Renamed to `JumpToButton`.